test/sharness/t0130-multinode.sh

```diff
@@ -79,7 +79,8 @@ run_advanced_test() {
 	run_random_dir_test
 
 	test_expect_success "shut down nodes" '
-		iptb stop
+		iptb stop ||
+			test_fsh cat .iptb/*/daemon.stderr
```

Comment:
In my experience: `tail -n +1` is better than cat as it also shows filenames.

I would go with `tail -n +1 .iptb/*/daemon.std*`